### ISSUE TITLE
BUG:The image is not visible

### ISSUE THREAD

**Issue by MuraliDharan7** [EternoSeeker/gameoflife#120]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The image section might having error which should be solved. The image is not visible only alt text is visible.

Please assign this task to me under GSSoC'24.

![gameoflife issue1](https://github.com/EternoSeeker/gameoflife/assets/130668745/1fc72b3d-7ea1-4f69-864f-5be88ddf62ce)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by EternoSeeker** [comment]
It is visible for me. Can you be a bit specific, in what case it's not working
